packages/kit/src/views/Discovery/pages/Dashboard/Welcome/Welcome.tsx

```
@@ -143,12 +146,15 @@ export function Welcome({ banner }: { banner: React.ReactNode }) {
         py="$6"
-        minHeight="$48"
+        $gtMd={{
+          minHeight: '$52',
+        }}
         $sm={{
           width: '100%',
         }}
       >
-        {banner || <DefaultTitle />}
+        {/* Show banner if provided, otherwise show DefaultTitle on non-native platforms */}
+        {banner || ((media.gtSm) && <DefaultTitle />)}
```

ESLint

[error] Replace `(media.gtSm)` with `media.gtSm`

(prettier/prettier)